src/libScilla/ScillaClient.cpp

```diff
@@ -78,6 +78,29 @@ void ScillaClient::Init() {
   }
 }
 
+bool ScillaClient::isScillaRuning() {
+  std::string cmdStr = "ps aux | awk '{print $2\"\\t\"$11}' | grep \"" +
+                       SCILLA_SERVER_BINARY + "\" | awk '{print $1}'";
```

Comment:
Yuck! but I guess this is idiomatic in our horrible, horrible world :-(

Reply:
Agreed; an alternative (a bit more effort though) would be to use Boost.Process to launch Scilla and use the launcher/handle as needed. Integrates nicely with Boost.Asio as well.

Reply:
I left it as humour, it was generated by github auto pilot, I personally would have used pidof, but copilot did a perfect job of emulating the programming style of the original author.